pkg/utils/defaulter/clusterresourceplacement.go

```diff
@@ -118,4 +112,11 @@ func SetDefaultsApplyStrategy(obj *fleetv1beta1.ApplyStrategy) {
 	if obj.WhenToTakeOver == "" {
 		obj.WhenToTakeOver = fleetv1beta1.WhenToTakeOverTypeAlways
 	}
+	// Always set the ServerSideApplyConfig when it is unset even if
+	// server-side apply itself might not be used, just for completeness reasons.
+	if obj.ServerSideApplyConfig == nil {
+		obj.ServerSideApplyConfig = &fleetv1beta1.ServerSideApplyConfig{
+			ForceConflicts: false,
+		}
+	}
```

Comment:
Will user see this? I am not sure if user should see their applyStrategy's serversideApply Config suddenly show up